.travis.yml

```diff
@@ -93,7 +106,8 @@ matrix:
                MATPLOTLIB_VERSION=1.5
 
         - os: linux
-          env: SETUP_CMD='test --coverage --remote-data=astropy -a "--mpl"'
+          stage: Initial tests
+          env: SETUP_CMD='test --coverage -a "--mpl"'
```

Comment:
Can you add back ``--remote-data=astropy`` - that should be stable and usually doesn't cause any issues

Reply:
I'm not sure if it's really stable. I restarted 5 tests that failed with an `URLError` or `BadGateWayException` just today. From my experience one in 4 builds chokes on `--remote-data=astropy`. That's okay for the second stage but probably much too often for one of the initial tests.

Reply:
Ah ok weird, in that case I agree. But this should definitely run during the second stage, with remote data and mpl (not just in cron)

Reply:
which other job would you prefer then in the first stage? We currently only have one without remote-data, the build with numpy 1.10 and without the optional dependencies.

Reply:
@bsipocz - just remove ``--mpl`` from this job, since it doesn't serve any purpose without ``--remote-data=astropy``.

Reply:
so basically we need to add another job back in the second stage?